docs/DeveloperGuide.md

```diff
@@ -17,7 +17,7 @@ Refer to the guide [_Setting up and getting started_](SettingUp.md).
 
 ### Architecture
 
-<img src="images/ArchitectureDiagram.png" width="450" />
+<img src="images/ArchetectureDiagram.png" width="450" />
```

Comment:
typo - should be ArchitectureDiagram. Remember to change your image filename too!

Reply:
okay changeed!